docs/using-nativeaot/compiling.md

```diff
@@ -99,6 +99,16 @@ public static int Answer()
 ```bash
 > dotnet publish /p:NativeLib=Static /p:SelfContained=true -r browser-wasm -c Debug /p:TargetArchitecture=wasm /p:PlatformTarget=AnyCPU /p:MSBuildEnableWorkloadResolver=false /p:EmccExtraArgs="-s EXPORTED_FUNCTIONS=_Answer -s EXPORTED_RUNTIME_METHODS=cwrap" --self-contained
 ```
+From Javascript, before calling your library the runtime must be initialised explicitly.  This is done with the following Javascript
+```js
+// Initialise the .Net runtime
+const corertInit = Module.cwrap('NativeAOT_StaticInitialization', 'number', []);
+corertInit();
+
```

Comment:
Is it actually needed to call it? The function only exists to trigger the global constructor.

Reply:
No it's not.  I've removed that from the documentation.  I though it was at one point.